diff --git a/node_modules/whatsapp-web.js/src/util/Constants.js b/node_modules/whatsapp-web.js/src/util/Constants.js
index 0fc3966..2302959 100644
--- a/node_modules/whatsapp-web.js/src/util/Constants.js
+++ b/node_modules/whatsapp-web.js/src/util/Constants.js
@@ -5,7 +5,8 @@ exports.WhatsWebURL = 'https://web.whatsapp.com/';
 exports.DefaultOptions = {
     puppeteer: {
         headless: true,
-        defaultViewport: null
+        defaultViewport: null,
+        args: ['--no-sandbox' ]
     },
     session: false,
     qrTimeoutMs: 45000,
